src/main/java/com/cgdp/library/controller/AuthorController.java

```diff
@@ -0,0 +1,32 @@
+package com.cgdp.library.controller;
+
+import com.cgdp.library.converters.AuthorDtoConverter;
+import com.cgdp.library.dto.CreateAuthorRequestDTO;
+import com.cgdp.library.entity.AuthorEntity;
+import com.cgdp.library.service.AuthorService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@AllArgsConstructor
+@RequestMapping("/authors")
+@RestController
+public class AuthorController {
+
+    private final AuthorService authorService;
+    private final AuthorDtoConverter authorDtoConverter;
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public AuthorEntity createAuthor(@RequestBody CreateAuthorRequestDTO createAuthorRequestDTO) {
```

Comment:
Entities classes are internal, they should not be exposed, either create a response DTO with relevant information to return, or simply create an `AuthorDTO` with the author data and return it